tasks/Ultimatum game (various versions).md

```diff
@@ -0,0 +1,41 @@
+# Summary
+>  One player, the proposer, is endowed with a sum of money and can split it with another player (the responder). The responder then can accept or reject the money. 
+
+# References
+> Task Description: https://en.wikipedia.org/wiki/Ultimatum_game
+
+# Stimuli
+## The visual components
+> Document or generate the visual stimuli seen by participants when performing this task. Provide screenshots or illustrations.
+
+## Materials for alternative versions of the experiment 
+>  competitive ultimatum game : there are many proposers and the responder can accept at most one of their offers: With more than three (naïve) proposers the responder is usually offered almost the entire endowment 
+  
+ > ultimatum game with tipping: a tip is allowed from responder back to proposer, a feature of the trust game, and net splits tend to be more equitable.
+   
+ > reverse ultimatum game: gives more power to the responder by giving the proposer the right to offer as many divisions of the endowment as they like. Now the game only ends when the responder accepts an offer or abandons the game, and therefore the proposer tends to receive slightly less than half of the initial endowment
+   
+ > Incomplete information ultimatum games: Some authors have studied variants of the ultimatum game in which either the proposer or the responder has private information about the size of the pie to be divided
+   
+ > The pirate game illustrates a variant with more than two participants with voting power
+   
+  
+# Procedure
+## Steps
+1. One player, the proposer, is endowed with a sum of money. The proposer is tasked with splitting it with another player, the responder. 
+2. Once the proposer communicates his decision, the responder may accept it or reject it. If the responder accepts, the money is split per the proposal; if the responder rejects, both players receive nothing. 
+3. Both players know in advance the consequences of the responder accepting or rejecting the offer
+4. Similarly, if the steps are interdependent, e.g., performance on one step influences what is seen on another step, this should be described in full detail.
+
+## Roles 
+> The proposer- endowed with a sum of money  
+> The responder- decides to accept or reject money from proposer.
+
+# Criteria
+## Performance calculation
+> Proposer Offer Amount
+  
+> Responder acceptance rate of proposer offer 
+
+## Incentives
+> This may vary depending on researchers running an experiment 
```

Comment:
I would say they are offered money for accepting, usually not given a showing up fee